client/components/SingleEvent/SingleEventDumb.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+class SingleEvent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {  }
+    }
+    render() { 
+        return ( 
+            <hr/>
```

Comment:
Is this just for another PR? Just want to make sure I'm not approving something you didn't mean to do.

Reply:
yeah it's just there, I'm still working on the events stuff but just wanted to get what I had up here because Justin wanted to work off of it

Reply:
Ok sounds good.